src/recorder.ts

```diff
@@ -36,11 +36,10 @@ export const getMp3MediaRecorder = (config: RecorderConfig): Promise<typeof Medi
 
         static isTypeSupported = (mimeType: string) => mimeType === MP3_MIME_TYPE;
 
-        constructor(stream: MediaStream) {
+        constructor(stream: MediaStream, audioContext?: AudioContext) {
```

Comment:
MediaRecorder constructor already has a second `options` argument. To keep this ponyfill fully compatible (and for extendability) I would like to keep the second argument as an object. Could you make the second argument an object with a property audioContext?